src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    List<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public TaskList(List<Task> existingTasks) {
+        this.tasks = (existingTasks == null ? new ArrayList<>() : existingTasks);
+    }
+
+    public int size() {
```

Comment:
Perhaps `size` could be named as `getSize` instead?

Reply:
Fixed! 🙂